dev/NumberBox/NumberBox.xaml

```diff
@@ -90,7 +92,7 @@
 
                         <Grid.RowDefinitions>
                             <RowDefinition Height="Auto" />
-                            <RowDefinition Height="Auto" />
+                            <RowDefinition Height="*" />
```

Comment:
Is this what fixed the height property?  How does this change fix the width property?

Reply:
This fixes the height issue, width worked fine. At least it does now.